.ci/rebuild_updated_recipes.py

```diff
@@ -12,7 +12,7 @@ def modified_recipes(branch='origin/master'):
     # with a bunch of fixes, e.g. disabled TTY, see:
     # https://stackoverflow.com/a/20128598/185510
     sh.contrib.git.fetch("origin", "master")
-    git_diff = sh.contrib.git.diff('--name-only', branch)
+    git_diff = sh.contrib.git.diff('--name-only --diff-filter=AM', branch)
```

Comment:
```suggestion
    git_diff = sh.contrib.git.diff('--name-only', '--diff-filter=AM', branch)
```

Reply:
Yeah, it's was a first guess. The docs on `sh.contrib.git.diff` are sparse...:-) Thanks

Reply:
ahah I couldn't help it :see_no_evil: